docs/manage/config-policies.rst

```diff
@@ -0,0 +1,263 @@
+   commands) tasks
+-  Allow priority overrides within policy constraints
+
+******************
+ Setting Policies
+******************
+
+Administrators can set Config Policies at either the cluster or workspace levels through the WebUI
+or CLI.
+
+WebUI
+=====
+
+Administrators can set Config Policies at both the cluster and workspace levels.
+
+.. tabs::
+
+   .. tab::
+
+      Cluster Config Policy
+
+      To set configuration policies at the cluster level:
+
+      #. Sign in to the Determined WebUI as a cluster administrator.
+      #. Navigate to **Config Policies**.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+   .. tab::
+
+      Workspace Config Policy
+
+      To set configuration policies at the workspace level:
+
+      #. Sign in to the Determined WebUI as a cluster or workspace administrator.
+      #. From your workspace, navigate to the **Config Policies** tab.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+CLI
+===
+
+Use the following commands to manage Task Config Policies via CLI:
+
+.. code:: bash
+
+   # List existing policies
+   det policy list
+
+   # Create a new policy
+   det policy create <policy_name> --config <policy_config_file.yaml>
+
+   # Update an existing policy
+   det policy update <policy_name> --config <updated_policy_config_file.yaml>
+
+   # Delete a policy
+   det policy delete <policy_name>
+
+*****************
+ Policy Examples
+*****************
+
+Below are some examples of Config Policies to help guide administrators.
+
+Limiting Resources
+==================
+
+Administrators can set constraints on resource usage, allowing them to manage how users allocate
+resources for workloads such as experiments, notebooks, tensorboards, shells, and commands. The two
+main configurable constraints are:
+
+-  ``resources.max_slots``: Limits the maximum number of slots (GPUs or CPUs) that can be used.
+-  ``resources.priority_limit``: Sets the priority limit for tasks.
+
+For Kubernetes resource managers, higher priority values indicate higher priority. For Agent
+resource managers, lower priority values indicate higher priority.
+
+**Example 1: Limiting GPU/CPU Slots and Priority for Experiments**
+
+The following example demonstrates how to set a maximum of 4 slots (GPUs or CPUs) and a priority
+limit of 15. This configuration applies to both Kubernetes and Agent resource managers, though the
+priority system behaves differently across the two:
+
+-  In Kubernetes, higher priority values indicate higher priority.
+-  In Agent resource managers, lower priority values indicate higher priority.
+
+.. code:: yaml
+
+   constraints:
+     resources:
+       max_slots: 4
+       priority_limit: 15
```

Comment:
Can we move priority limit out of resources? We don't have time to include it in resources after all.